src/setup_payload/SetupPayload.h

```diff
@@ -160,7 +161,7 @@ struct OptionalQRCodeInfoExtension : OptionalQRCodeInfo
     uint64_t uint64;
 };
 
-bool IsCHIPTag(uint8_t tag);
+bool IsCommonTag(uint8_t tag);
 bool IsVendorTag(uint8_t tag);
```

Comment:
Could we make these static SetupPayload methods instead of global ones?

Reply:
IsCommonTag() is being used in another compilation unit (QRCodeSetupPayloadParser.cpp), unfortunately.

Reply:
Yes, but if it's a class static method that's not a problem.

Reply:
Ah, put it in the SetupPayload class!?. Got it.